lib/api/policy.go

```diff
@@ -93,6 +93,10 @@ type PolicySpec struct {
 	// 	deployment != "dev"
 	// 	! has(label_name)
 	Selector string `json:"selector" validate:"selector"`
+
+	// Indicates, if True, that the rules in this policy should be applied before any data plane
+	// connection tracking, and that packets allowed by these rules should not be tracked.
+	Untracked bool `json:"untracked,omitempty" validate:"omitempty"`
```

Comment:
Very happy to make that change, but should I wait for a view from @robbrockbank as well?